File: src/main/java/net/dontdrinkandroot/gitki/wicket/event/AbstractEvent.java
package net.dontdrinkandroot.gitki.wicket.event;

import org.apache.wicket.ajax.AjaxRequestTarget;

/**
 * @author Philip Washington Sorst <philip@sorst.net>
 */
public class AbstractEvent
{
    private AjaxRequestTarget target;

    public AbstractEvent()
    {
    }

    public AbstractEvent(AjaxRequestTarget target)
    {
        this.target = target;
    }

    public AjaxRequestTarget getTarget()
    {
        return this.target;
    }
}

File: src/main/kotlin/net/dontdrinkandroot/gitki/wicket/event/AbstractEvent.kt
package net.dontdrinkandroot.gitki.wicket.event

import org.apache.wicket.ajax.AjaxRequestTarget

open class AbstractEvent {

    var target: AjaxRequestTarget? = null
        private set

    constructor()

    constructor(target: AjaxRequestTarget?) {
        this.target = target
    }
}